### PR TITLE
created tabs with arrows

### PR DESCRIPTION
**Pull request by Emily9023**
colors are not finished!! I couldn't find where the hex codes were but will be asking the team once I get the chance (prolly at the meeting)

I feel like there is definitely a better way of doing this so if anyone has an idea, pls shoot me a msg or comment on this pull request!!

### CODE REVIEW

**Comment by jameskokoska** [comment]
When you get a chance please provide a Storybook

As per our previous convo, 

I always reference `Confetti.stories.jsx` it's a good example
```js
import React from 'react';

import { Confetti } from './Confetti';

export default {
  title: 'Confetti',
  component: Confetti,
};

export const ConfettiStory = (args) => <Confetti {...args} />;
ConfettiStory.storyName = 'Confetti';
ConfettiStory.args = {
  animate: true,
};
```

To create a story, create a file with `.stories.jsx` at the end
To view the story, in client, run `yarn storybook`

File: client/src/components/tabs/tabs.scss [hunk 1]
```diff
@@ -0,0 +1,115 @@
+@import '../../scssStyles/mixins.scss';
+
+.Tabs-desktop {
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.scss:3]: Stick to lower case with dashes

File: client/src/components/tabs/tabs.scss [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+.tabs-center {
+  text-align: center;
+  margin: auto;
+}
+
+.inline-block {
+  position: inline-block;
+  float: left;
+}
+.test {
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.scss:107]: Remove unused CSS classes

File: client/src/components/tabs/tabs.scss [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  border: 2px solid var(--white);
+  border-bottom: none;
+}
+
+.tabs-center {
+  text-align: center;
+  margin: auto;
+}
+
+.inline-block {
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.scss:103]: Avoid generic CSS names

File: client/src/components/tabs/tabs.scss [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+.tab-content {
+  background-color: var(--light-purple);
+  width: 100%;
+  height: 75%;
+  position: relative;
+}
+
+.relative {
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.scss:41]: Avoid generic CSS names

File: client/src/components/tabs/tabs.scss [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+  position: relative;
+}
+
+.footer {
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.scss:45]: avoid generic CSS names

File: client/src/components/tabs/tabs.scss [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+  position: absolute;
+  bottom: 60px;
+  width: 100%;
+}
+
+/* ------- BUTTONS ------- */
+.arrow-btn {
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.scss:52]: Avoid generic CSS names

File: client/src/components/tabs/tabs.scss [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+  background-color: var(--yellow);
+  color: black;
+}
+
+.nav-btn {
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.scss:57]: Avoid generic CSS names

File: client/src/components/tabs/tabs.scss [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+  border-bottom-right-radius: 0px;
+  border-bottom-left-radius: 0px;
+  margin-bottom: 0;
+}
+
+.btn-left {
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.scss:63]: Avoid generic CSS names

File: client/src/components/tabs/tabs.scss [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+  position: absolute;
+  left: 15px;
+}
+
+.btn-right {
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.scss:68]: Avoid generic CSS names

File: client/src/components/tabs/tabs.scss [hunk 10]
```diff
@@ -0,0 +1,115 @@
...
+
+.tab-main {
+  width: 100%;
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  bottom: 0px;
+  left: 0px;
+}
+.icon_container {
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.scss:24]: use lower case with dashes
and avoid generic CSS names

File: client/src/components/tabs/tabs.jsx [hunk 11]
```diff
@@ -0,0 +1,257 @@
...
+              )}
+            </div>
+            <div className="btn-right">
+              {tabs.indexOf(currentTab) < tabs.length - 1 && (
+                <Button
+                  label={
+                    <div className="tabs-block">
+                      <div className="inline-block tabs-link">{'Next'}</div>
+                      <div className="inline-block icon_container">
+                        <svg
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.jsx:174]: Download and import the file locally
Use React Import asset

File: client/src/components/tabs/tabs.jsx [hunk 12]
```diff
@@ -0,0 +1,257 @@
...
+
+          <div className="inline-block footer">
+            <div className=" btn-left">
+              {tabs.indexOf(currentTab) > 0 && (
+                <Button
+                  label={
+                    <div>
+                      <div className="inline-block tabs-link ">{'Back'}</div>
+                      <div className="inline-block icon_container ">
+                        <svg
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.jsx:144]: Download and import the file locally

File: client/src/components/tabs/tabs.jsx [hunk 13]
```diff
@@ -0,0 +1,257 @@
...
+  tabs.map((tab, index) => {
+    dropDownItemsArr[index] = tab.tabTitle;
+  });
+  const items = ['Option 1', 'Option 2', 'Option 3'];
+
+  return (
+    <div className="tab-main">
+      <div className="tab-container tabs-center">
+        <div className="tabs-center relative">
+          {/* <Dropdown
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.jsx:125]: Remove unused code

File: client/src/components/tabs/tabs.jsx [hunk 14]
```diff
@@ -0,0 +1,257 @@
...
+};
+
+const TabsMobile = ({ tabs }) => {
+  var dropDownItemsArr = [];
+  const [currentTab, setCurrentTab] = useState(tabs[0]);
+
+  tabs.map((tab, index) => {
+    dropDownItemsArr[index] = tab.tabTitle;
+  });
+  const items = ['Option 1', 'Option 2', 'Option 3'];
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.jsx:119]: Remove unused code

File: client/src/components/tabs/tabs.jsx [hunk 15]
```diff
@@ -0,0 +1,257 @@
...
+              )}
+            </div>
+            <div className="btn-right">
+              {tabs.indexOf(currentTab) < tabs.length - 1 && (
+                <Button
+                  label={
+                    <div className="tabs-block">
+                      <div className="inline-block tabs-link">{'Next'}</div>
+                      <div className="inline-block icon_container">
+                        <svg
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.jsx:82]: download and import svg asset

File: client/src/components/tabs/tabs.jsx [hunk 16]
```diff
@@ -0,0 +1,257 @@
...
+
+          <div className="inline-block footer">
+            <div className=" btn-left">
+              {tabs.indexOf(currentTab) > 0 && (
+                <Button
+                  label={
+                    <div className="tabs-block">
+                      <div className="inline-block tabs-link ">{'Back'}</div>
+                      <div className="inline-block icon_container ">
+                        <svg
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.jsx:52]: download and import asset

File: client/src/components/tabs/tabs.jsx [hunk 17]
```diff
@@ -0,0 +1,257 @@
...
+        <TabsDesktop tabs={tabs} />
+      </div>
+      <div className="Tabs-mobile">
+        <TabsMobile tabs={tabs} />
+      </div>
+    </div>
+  );
+};
+
+const TabsDesktop = ({ tabs }) => {
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.jsx:20]: I think keeping the same format for both web and mobile should be okay
It's generally bad practice to load/unload component from resizing
The desktop format should work okay on mobile no?

File: client/src/components/tabs/tabs.jsx [hunk 18]
```diff
@@ -0,0 +1,257 @@
...
+  return (
+    <div className="tab-main">
+      <div className="tab-container tabs-center">
+        <div className="tabs-center relative">
+          {tabs.map((tab, index) => {
+            {
+              var css_class = tab == currentTab ? 'nav-btn active_link' : 'nav-btn non_active_link';
+            }
+            return (
+              <Button
```
Comment (jameskokoska) [inline client/src/components/tabs/tabs.jsx:32]: You can use the `<ButtonSelector>` component that was recently made to handle the selection and styles for you

**Review by jameskokoska — CHANGES_REQUESTED** [review]
Looks good, maybe a few style changes and formatting fixes
Maybe add some rounded corners to the parent container
If the inner container (tab content) overflows, make it scrollable
Maybe make it a bit more obvious which tab is selected (use the ButtonSelector component)
image

**Comment by jameskokoska** [comment]
image

**Comment by jameskokoska** [comment]
@natapokie Could you change the colors of the arrows for me
I don't have illustrator on my laptop
```js
import ArrowRight from '../../assets/steps/arrow-right-solid.svg';
import ArrowLeft from '../../assets/steps/arrow-left-solid.svg';
```

**Review by jameskokoska — APPROVED** [review]

**Comment by jameskokoska** [comment]
@Emily9023 Feel free to review these changes as well
You don't need to submit a proper Review since you own this PR

**Review by natapokie — APPROVED** [review]
Looks really good!! The tabs are so cool

**Comment by natapokie** [comment]
Also @jameskokoska, I found out recently that you can actually change the colour of an svg file inside the file, so you don't need to use illustrator!

If you add `fill="#hexcode"` to the `<path` or `<polygon`, etc. you can colour the svg file! I just thought it was really cool, and wanted to share this!
image

**Comment by jameskokoska** [comment]
oh ya that's useful
I just didn't do it on the ones from font awesome because that tag didn't exist to begin with
but good to know you can add it in!

**Pull request merged**